oeps/oep-0017-feature-toggles.rst

```diff
@@ -0,0 +1,809 @@
+|   |                 |                                     |   `overriding values`_ in tests,     |
+|   |                 |                                     |   which the framework can adapt.     |
++---+-----------------+-------------------------------------+--------------------------------------+
+| 5 | Auditability    | Operators and teams should be able  | **Not natively, but...**             |
+|   |                 | to tell the who, what, and when of  |   Can view history in                |
+|   |                 | toggle changes.                     |   django_admin_log table for edits   |
+|   |                 |                                     |   made via Django admin. Any new     |
+|   |                 |                                     |   models added by the framework      |
+|   |                 |                                     |   should also support auditability.  |
++---+-----------------+-------------------------------------+--------------------------------------+
+| 6 | Performance     | The value of a toggle should be     | **Yes, but...**                      |
+|   |                 | cached so it is not repeatedly      |   Cached using `Django cache`_, but  |
+|   |                 | retrieved from storage.             |   the framework also needs to cache  |
+|   |                 |                                     |   in a request-specific cache to     |
+|   |                 |                                     |   avoid repeated hits to Memcached.  |
++---+-----------------+-------------------------------------+--------------------------------------+
+| 7 | Toggle types:   | The 3 necessary toggle types are    | **Yes, but...**                      |
+|   |                 | supported and easy to use by edX    |   Waffle's Switch_ class supports the|
+|   | - Switch        | developers.                         |   "Switch" toggle type. Waffle's     |
+|   | - Rollout       |                                     |   Flag_ class supports the "Rollout" |
+|   | - Group         |                                     |   toggle type. However, since edX    |
+|   |                 |                                     |   (currently) does not store course  |
+|   |                 |                                     |   and organization relationships as  |
+|   |                 |                                     |   `Django groups`_, the framework    |
+|   |                 |                                     |   must provide support for the       |
+|   |                 |                                     |   "Group" toggle type.               |
++---+-----------------+-------------------------------------+--------------------------------------+
+| 8 | Non-collision   | Feature toggles created by          | **No.**                              |
+|   |                 | independent teams should not collide|   The framework must support         |
```

Comment:
You use the following text below, which I found added some clarity: `The namespace should be unique
+to each Django app so it doesn't collide with other installed apps in the system.`

Reply:
I am suggesting that this be further clarified, because I was confused until I got to the statement that I added above.  Thanks.

Reply:
Instead of duplicating the statement, I added a note above the table to indicate there's further detail that follows the table.